### ISSUE TITLE
getting the pseudolabels

### ISSUE THREAD

**Issue by hosea7456** [yukimasano/self-label#17]
Hi, I wonder if you have the code for getting the pseudolabels of training stage? As you recommend:

"As we show in the paper, the pseudolabels we generate from our training can be used to quickly train a neural network with regular cross-entropy. Moreover they seem to correctly group together similar images. "

I want to get the pseudolabels as I train the model on a new dataset.
Thank you!

**Comment by yukimasano** [comment]
Hi,

Please just uncomment this line to save the pseudolabels at every iteration:
https://github.com/yukimasano/self-label/blob/master/main.py#L61 
:) 

Also, in case you haven't seen it: here you can interactively go through some clusters (after training):
https://www.robots.ox.ac.uk/~vgg/research/self-label/ 

Cheers
